scripts/pre_commit_linter.py

```diff
@@ -215,6 +216,7 @@
 
 import isort             # pylint: disable=wrong-import-position
 import pycodestyle       # pylint: disable=wrong-import-position
+from pyjsparser import PyJsParser  # pylint: disable=wrong-import-position
```

Comment:
import pyjsparser

and then use pyjsparser.PyJsParser() below. Always import at module level, not class level.

Reply:
Done.